lib/src/utils/sanitizer.dart

```diff
@@ -99,18 +112,24 @@ Future<String> sanitizeContent(
               },
           },
         SanitizerTarget.emojis => match.group(2)!,
+        SanitizerTarget.commands => match.namedGroup('commandName')!,
       };
 
   String prefix(SanitizerTarget target) => switch (target) {
         SanitizerTarget.users || SanitizerTarget.roles => '@',
         SanitizerTarget.everyone => '@$_whitespaceCharacter',
         SanitizerTarget.channels => '#',
         SanitizerTarget.emojis => '',
+        SanitizerTarget.commands => '</',
```

Comment:
Should just be `/` - the prefix function is used to provide a visual representation of what the text _would_ look like, if it wasn't sanitized.

The exception to this is emojis, which are rendered as `:emoji_name:`, since Discord users tend to know that emojis are surrounded by colons and we don't have a good way to convert them to text otherwise.

Also, I just noticed the line above should be `=> ':'` and not `=> ''`, could you add that to this PR?